docs/skopeo-standalone-verify.1.md

```diff
@@ -8,7 +8,9 @@ skopeo\-standalone\-verify - Verify an image signature.
 
 ## DESCRIPTION
 
-Verify a signature using local files, digest will be printed on success.
+Verify a signature using local files, digest will be printed on success. This is primarily a debugging tool, or useful for special cases,
```

Comment:
```suggestion
Verify a signature using local files; the digest will be printed on success. This is primarily a debugging tool, useful for special cases,
```

Reply:
A couple of small suggestions for restructuring slightly, otherwise LGTM

Reply:
Thanks @TomSweeneyRedHat appreciate the kind feedback; have amended both `docs/skopeo-standalone-sign.1.md` and `docs/skopeo-standalone-verify.1.md` using the same convention. Cheers, Dave